.vscode/settings.json

```diff
@@ -8,5 +8,6 @@
   "workbench.colorCustomizations": {
     "titleBar.activeBackground": "#2596be",
     "titleBar.inactiveBackground": "#004a72"
-  }
+  },
+  "editor.experimental.stickyScroll.enabled": true
```

Comment:
do we have to enable stickyScroll?

Reply:
I just did it for myself, it's a fun feature